alpaca_trade_api/stream.py

```diff
@@ -565,12 +565,15 @@ def _cast(self, msg_type, msg):
 
     async def _dispatch(self, msg):
         msg_type = msg.get('T')
-        symbol = msg.get('S')
         if msg_type == 'n':
-            handler = self._handlers['news'].get(
-                symbol, self._handlers['news'].get('*', None))
-            if handler:
-                await handler(self._cast(msg_type, msg))
+            symbols = msg.get('symbols', [])
+            if not symbols:
+                symbols.append('*')
+
+            for symbol in symbols:
+                handler = self._handlers['news'].get(symbol)
```

Comment:
Have you tested this? I don't think it works well with a single `*` subscription. In that case `self._handlers["news"]` will have a single `*` -> handler entry, which won't be returned for a single `symbol`.

Reply:
Oops, will it add it back. Good catch!

Reply:
Updated!